pages/linux/netselect-apt.md

```diff
@@ -0,0 +1,20 @@
+# netselect-apt
+
+> Create sources.list for Debian mirror with the lowest latency.
```

Comment:
Perhaps this will read better?
```suggestion
> Create a `sources.list` file for a Debian mirror with the lowest latency.
```

Reply:
Ahh! right a file name should have been in back ticks.